addons/payment/models/account_move.py

```diff
@@ -23,6 +28,17 @@ def _compute_authorized_transaction_ids(self):
                 lambda tx: tx.state == 'authorized'
             )
 
+    @api.depends('transaction_ids')
+    def _compute_amount_paid(self):
+        """ Sum all the transaction amount for which state is in 'pending', 'authorized' or 'done'
+        """
+        for invoice in self:
+            invoice.amount_paid = sum(
+                invoice.transaction_ids.filtered(
+                    lambda tx: tx.state in ('authorized', 'done')
```

Comment:
@AntoineVDV Nice spot, fixed !

Reply:
@abz-odoo Actually, I think it will be better to add 'pending' in the list than remove it from the docstring :)

```suggestion
                    lambda tx: tx.state in ('pending', 'authorized', 'done')
```